Lombiq.Tests.UI.Samples/xunit.runner.json

```diff
@@ -2,5 +2,5 @@
   "$schema": "https://xunit.net/schema/current/xunit.runner.schema.json",
   "parallelizeAssembly": false,
   "parallelizeTestCollections": true,
-  "maxParallelThreads": 3
```

Comment:
You can leave `"maxParallelThreads": 3` since this was here before. I wanted you to remove it from the other project because we set the value there from a GH action.